test/input_schema.test.ts

```diff
@@ -208,6 +223,47 @@ describe('input_schema.json', () => {
             );
         });
 
+        it('should throw error on empty enum array', () => {
+            const schema = {
+                title: 'Test input schema',
+                type: 'object',
+                schemaVersion: 1,
+                properties: {
+                    myField: {
+                        title: 'Field title',
+                        type: 'string',
+                        description: 'Some description ...',
+                        enum: [],
+                    },
+                },
+            };
+
+            expect(() => validateInputSchema(validator, schema)).toThrow(
+                'Input schema is not valid (Field schema.properties.myField.enum.enum must NOT have fewer than 1 items)',
```

Comment:
I have no idea why there is `.enum.enum` here, but AJV codebase is such a mess and I don't have the motivation to debug it deeper